### ISSUE TITLE
missing 4th question from the server side of this tool

### ISSUE THREAD

**Issue by GoogleCodeExporter** [soimort/codejam-commandline#20]
```
What steps will reproduce the problem?
try to download the 4th problem in the qualification round

What is the expected output? What do you see instead?
file should be downloaded to input directory instead I get an error message 
that .

What version of the product are you using? On what operating system?
windows 7
python 27
V1.0B2

Addition information:
Perhaps this was not reported because of issue #18


```

Original issue reported on code.google.com by `veltzerd...@gmail.com` on 15 Apr 2012 at 2:28

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
I have no idea why I'm not getting emails when somebody adds a new issue, I 
guess I'll have to check the project configuration.

We are now aware of this problem, a fix is on the way. Problem D (and B and C) 
became innacessible from the commandline because problem A has no large input, 
which confused the tool.
```

Original comment by `jor...@google.com` on 9 Jun 2012 at 1:28
- Changed state: **Started**
- Added labels: **Priority-Critical**, **OpSys-All**, **Usability**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Issue 18 has been merged into this issue.
```

Original comment by `jor...@google.com` on 9 Jun 2012 at 1:30

**Comment by GoogleCodeExporter** [comment]
```
Issue 17 has been merged into this issue.
```

Original comment by `jor...@google.com` on 9 Jun 2012 at 1:31

**Comment by GoogleCodeExporter** [comment]
```
This issue has been fixed in r11, which is available in version 1.1-beta1 in 
the downloads.
```

Original comment by `jor...@google.com` on 9 Jun 2012 at 2:18
- Changed state: **Fixed**
